tests/libnode/node.go

```diff
@@ -333,6 +333,16 @@ func GetNodeWithHugepages(virtClient kubecli.KubevirtClient, hugepages k8sv1.Res
 	return nil
 }
 
+func NodeNameWithHandler() string {
```

Comment:
```suggestion
func GetNodeNameWithHandler() string {
```

For consistency with other helpers found in this file?

Reply:
Not a big deal but consider this suggestion, think it got lost somewhere during the review rounds

Reply:
oh, so sorry I've missed it. I thought that I have applied @0xFelix's request. Here it is now: https://github.com/kubevirt/kubevirt/pull/13170